### DISCUSSION TITLE
Only snapshot

### DISCUSSION THREAD

**Discussion by 12nick12** [oetiker/znapzend#604] [Q&A] [answered]
Is there anyway to only take snapshots and not send them anywhere?

**Comment by trijenhout** [comment]
give a false ip or somthing will work, do you need auto destroy?

**Comment by 12nick12** [comment]
yes, I would like to to keep up with my retention policy.

**Comment by oetiker** [comment, marked as answer]
just don't configure a DST and you should be all set
